scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -191,8 +191,8 @@ task GetColumnNames {
     }
 
     ## set some output files
-    String vcf_files_column_name_output = "vcf_files_column_name.txt"
-    String vcf_index_files_column_name_output = "vcf_index_files_column_name.txt"
+    String vcf_files_column_name_output_file = "vcf_files_column_name.txt"
```

Comment:
miniwdl check flagged an error here. The same name was being used for an input and an output to this task. I wanted to use vcf_files_column_name_output as the name of the output so I renamed this variable.

Reply:
Yay miniwdl. This error has bitten me before and is not fun to figure out when womtool doesn't report it.